runtime/master/src/main/java/org/apache/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -113,13 +114,29 @@
   private final Server metricServer;
   private final MetricStore metricStore;
 
+  /**
+   * Constructor.
+   *
+   * @param scheduler                the scheduler implementation.
+   * @param containerManager         the container manager, in charge of the available containers.
+   * @param metricMessageHandler     the handler for metric messages.
+   * @param masterMessageEnvironment message environment for the runtime master.
+   * @param metricManagerMaster      metric manager master.
+   * @param clientRPC                the RPC channel to communicate with the client.
+   * @param planStateManager         the manager that keeps track of the plan state.
+   * @param jobId                    the Job ID, provided by the user.
+   * @param dbAddress                the DB Address, provided by the user.
+   * @param dbId                     the ID for the given DB.
```

Comment:
Does runtime master should hold these XGBoost specific variables? Maybe it would be better to extract these variables to other classes likeXGBoostMaster? or these variables are not related to XGBoost?

Reply:
These are actually related to the metrics data, which can be collected with appropriate information about the DB, provided through the nemo job options. I would say that these are not that XGBoost-specific.